pepper-apis/dsm-core/src/test/resources/elastic/lmsActivitiesSharedLearningEligible.json

```diff
@@ -44,6 +44,12 @@
     "SOMATIC_CONSENT_ADDENDUM_TUMOR" : true,
     "questionType" : "BOOLEAN"
     },
+    {
```

Comment:
Because there was no SOMATIC_CONSENT_TUMOR value in the json, the test could not reset the answer.  Adding the block here means that the test can manipulate the value as needed.